index.html

```diff
@@ -39,257 +42,263 @@
-            </path>
-          </svg></a>
-      </li>
-    </ul>
-    <p>
-      Обратная связь:
-      <a href="mailto:mail@htmlacademy.ru">mail@htmlacademy.ru</a>
-    </p>
-    <p>
-      Разработано -
-      <a href="https://htmlacademy.ru">htmlacademy.ru</a>
-    </p>
+  <footer class="page-footer">
+    <div>
+      <ul class="navigation footer-nav">
+        <li><a class="footer-technomart-link" href="">ТЕХНОМАРТ</a></li>
+        <li><a href="">Компания</a></li>
+        <li><a href="">Новости</a></li>
+        <li><a href="catalog.html">Каталог</a></li>
+        <li><a href="">Доставка</a></li>
+        <li><a href="">Контакты</a></li>
+      </ul>
+      <address class="footer-address">г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</address>
+      <p><a class="footer-phone" href="tel:+7(812)555-05-55">+7 (812) 555-05-55</a></p>
+      <ul class="navigation footer-features">
+        <li><a href="">Материалы</a></li>
+        <li><a href="">Техника</a></li>
+        <li><a href="">Инструмент</a></li>
+        <li><a href="">Спецпредложения</a></li>
+      </ul>
+    </div>
+    <div>
+      <p class="footer-copyright">
+        © 2010–2019 Компания «Техномарт»
+        Все права защищены
+      </p>
+      <ul>
+        <li>
+          <a class="social-button" href="https://vk.com/" target="_blank" rel="noopener nofollow" title="VK"><span
+              class="sr-only"></span><svg width="24" height="24" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg">
+              <path
+                d="M15.83 25.176h1.63s.45-.09.723-.362c.27-.27.18-.724.18-.724s0-2.17.996-2.533c.994-.362 2.26 2.08 3.708 3.076.995.724 1.81.543 1.81.543l3.618-.09s1.9-.09.995-1.628c-.09-.09-.542-1.086-2.713-3.166-2.262-2.17-1.99-1.81.814-5.428 1.72-2.262 2.353-3.62 2.172-4.16-.18-.544-1.447-.453-1.447-.453h-4.16s-.272 0-.544.09c-.18.09-.36.453-.36.453s-.634 1.72-1.54 3.257c-1.808 3.076-2.532 3.257-2.894 3.076-.724-.452-.543-1.81-.543-2.804 0-2.985.452-4.252-.905-4.614-.452-.09-.814-.18-1.9-.18-1.447 0-2.714 0-3.438.36-.452.272-.814.725-.633.815.27 0 .905.18 1.176.633.452.543.452 1.9.452 1.9s.27 3.528-.543 3.98c-.543.27-1.357-.27-2.985-3.166-.814-1.447-1.447-3.076-1.447-3.076s.09-.27-.18-.452-.634-.27-.634-.27l-3.89.09s-.543 0-.814.27c-.18.27 0 .724 0 .724S5.608 18.483 9.045 22.1c3.166 3.258 6.785 3.077 6.785 3.077z">
+              </path>
+            </svg></a>
+        </li>
+        <li>
+          <a class="social-button" href="https://www.facebook.com/" target="_blank" rel="noopener nofollow"
+            title="facebook"><span class="sr-only"></span><svg width="24" height="24" viewBox="0 0 32 32"
```

Comment:
Круто, что добавила `title`, это забота о пользователе. Для единообразия давай писать все тайтлы с большой буквы
```suggestion
            title="Facebook"><span class="sr-only"></span><svg width="24" height="24" viewBox="0 0 32 32"
```

Reply:
Исправила тайтлы и добавила тайтлы на кнопки.